cmd/plural/ops_test.go

```diff
@@ -0,0 +1,124 @@
+package main_test
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"gopkg.in/yaml.v2"
+
+	plural "github.com/pluralsh/plural/cmd/plural"
```

Comment:
`plural` prefix can be removed.

Reply:
strange but code can't be compiled without this prefix